### PR TITLE
feat: v0 model operation node

### PR DESCRIPTION
**Pull request by shawnyama**
# Description

Model selection + config node

https://user-images.githubusercontent.com/28237258/235509225-adb762df-4d72-4dc2-8204-539d0fb96424.mov

### CODE REVIEW

File: packages/client/hmi-client/src/page/project/tera-project.vue [hunk 1]
```diff
@@ -646,7 +647,7 @@ section {
 
 :deep(span.p-dropdown-trigger-icon.pi.pi-chevron-down) {
 	color: var(--text-color-subdued);
-}
+} */
```
Comment (mwdchang) [inline packages/client/hmi-client/src/page/project/tera-project.vue]: What and why are we commenting out a section here? This change seems unrelated?

It would be good to keep PRs as single purpose as possible - while it is convenient to fix things as you come across it does make tracking things extra hard, especially since we are squashing commits it makes reverting commits and cherry-picking commits very difficult.

Comment (shawnyama) [reply packages/client/hmi-client/src/page/project/tera-project.vue]: I deleted those styles later, it was making the dropdown in the model-node look bad

**Review by mwdchang — APPROVED** [review]
